frappe/utils/html_utils.py

```diff
@@ -42,54 +43,23 @@ def clean_html(html):
 			"tbody",
 			"td",
 			"tr",
-		],
+		},
 		attributes=[],
-		styles=["color", "border", "border-color"],
+		css_sanitizer=CSSSanitizer(allowed_css_properties=["color", "border", "border-color"]),
```

Comment:
can delete this line, since "style" attribute get's stripped because "style" attribute is not the part of allowed attributes. 

Useful while cleaning "email html" since [stye attribute](https://github.com/frappe/frappe/pull/21782/files#diff-ce46a98b68c5bf0aa23363d93174e1056b80d0a0dce065344fa3484f3622910aR129) is allowed there.
```suggestion
```